lib/winnow/model.rb

```diff
@@ -38,6 +38,9 @@ def search(all_params)
             scoped = scoped.where(name => val)
           elsif contains_scopes.include?(name.to_s)
             column = name.to_s.gsub("_contains", "")
+            scoped = scoped.where("#{table_name}.#{column} like ?", "%#{value}%")
+          elsif starts_with_scopes.include?(name.to_s)
+            column = name.to_s.gsub("_starts_with", "")
             scoped = scoped.where("#{table_name}.#{column} like ?", "#{value}%")
           elsif scoped.respond_to?(name)
```

Comment:
@ianlenehan this section isn't hit for scopes that match above. That's why the spec is still failing - because `name` is a column on the `users` table.